README.md

```diff
@@ -102,26 +102,35 @@ The release process is semi-automated.
 > - Create a draft GitHub release,
 > - Upload the Helm chart tarball to the GitHub release.
 
-1. Create a tag for the new release:
+1. Open the [tests GitHub Actions workflow][tests-workflow]
+   and verify that it succeeds on the master branch.
+2. Run govulncheck:
+   ```bash
+   go install golang.org/x/vuln/cmd/govulncheck@latest
+   govulncheck -v ./...
+   ```
```

Comment:
The govulncheck action is broken

https://github.com/jetstack/jetstack-secure/actions/workflows/govulncheck.yaml

So until we fix that, we have to run govulncheck manually before releasing.

Reply:
We can fix that in another PR: 
* #634